### PR TITLE
When using enablerepo in config.toml, allow spaces and dupulication

### PR DESCRIPTION
**Pull request by a2atsu**
## What did you implement:

Closes #XXXXX

## How did you implement it:

define removeDup function and invoke it from Load(pathToToml, keyPass string) function.

## How can we verify it:
in config.toml, 

enablerepo = "  base,            updates, updates                "

## Todos:
You don't have to satisfy all of the following.

- [ ] Write tests
- [ ] Write documentation
- [x] Check that there aren't other open pull requests for the same issue/feature
- [x] Format your source code by `make fmt`
- [x] Pass the test by `make test`
- [ ] Provide verification config / commands
- [x] Enable "Allow edits from maintainers" for this PR
- [ ] Update the messages below

***Is this ready for review?:*** NO  
***Is it a breaking change?:*** NO

### CODE REVIEW

**Comment by a2atsu** [comment]
epel 対応入れる前に、config.toml内にenablerepoでbaseとupdatesを記載する際にスペースが入ったりした場合でも、読み込めるようにした（念のため、updatesを２回書いたりした場合の重複対応もした）

File: config/tomlloader.go [hunk 1]
```diff
@@ -158,22 +170,28 @@ func (c TOMLLoader) Load(pathToToml, keyPass string) error {
...
 		}
-
+		repos := ""
 		s.Enablerepo = v.Enablerepo
 		if len(s.Enablerepo) == 0 {
 			s.Enablerepo = d.Enablerepo
 		}
 		if len(s.Enablerepo) != 0 {
 			for _, repo := range strings.Split(s.Enablerepo, ",") {
+                                repo = strings.TrimSpace(repo)
```
Comment (kotakanbe) [inline config/tomlloader.go]: format this source code before sending P/R.
```
make fmt
```

File: config/tomlloader.go [hunk 2]
```diff
@@ -158,22 +170,28 @@ func (c TOMLLoader) Load(pathToToml, keyPass string) error {
 				s.Optional = append(s.Optional, dkv)
 			}
 		}
-
+		repos := ""
```
Comment (kotakanbe) [inline config/tomlloader.go]: Move this line into `if block` to make the scope of variables locally

File: config/tomlloader.go [hunk 3]
```diff
@@ -158,22 +170,28 @@ func (c TOMLLoader) Load(pathToToml, keyPass string) error {
...
 				case "base", "updates":
-					// nop
+					repos = repos + "," + repo		
 				default:
 					return fmt.Errorf(
 						"For now, enablerepo have to be base or updates: %s, servername: %s",
 						s.Enablerepo, name)
 				}
 			}
+			if strings.HasPrefix(repos,","){
```
Comment (kotakanbe) [inline config/tomlloader.go]: I think this block is unnecessary.

File: config/tomlloader.go [hunk 4]
```diff
@@ -27,6 +27,18 @@ import (
 	"github.com/future-architect/vuls/contrib/owasp-dependency-check/parser"
 )
 
+func removeDup(repos []string) (uniq []string) {
+	results := []string{}
```
Comment (kotakanbe) [inline config/tomlloader.go]: remove results

File: config/tomlloader.go [hunk 5]
```diff
@@ -27,6 +27,18 @@ import (
...
+	seen := map[string]bool{}
+	for _, s := range repos {
+		if !seen[s] {
+			seen[s] = true
+			results = append(results, s)
```
Comment (kotakanbe) [inline config/tomlloader.go]: uniq = append(uniq, s)

File: config/tomlloader.go [hunk 6]
```diff
@@ -27,6 +27,18 @@ import (
...
+		}
+	}
+	return results
```
Comment (kotakanbe) [inline config/tomlloader.go]: return results
to 
return

File: config/tomlloader.go [hunk 7]
```diff
@@ -158,22 +170,25 @@ func (c TOMLLoader) Load(pathToToml, keyPass string) error {
 				s.Optional = append(s.Optional, dkv)
 			}
 		}
-
+		//		repos := ""
```
Comment (kotakanbe) [inline config/tomlloader.go]: remove this comment

**Pull request closed without merging**